.github/workflows/verification-tests.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Verification Tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron:  "47 0 * * *"
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: "opensafely-core/setup-action@v1"
+        with:
+          python-version: "3.12"
+          install-just: true
+
+      - name: Install venv
+        run: just devenv
+
+      - name: Run verification tests
+        env:
+          GITHUB_TOKEN_TESTING: ${{ secrets.OPENSAFELY_GITHUB_TESTING_ORG_PAT }}
+          SECRET_KEY: 12345
+          SOCIAL_AUTH_GITHUB_KEY: test
+          SOCIAL_AUTH_GITHUB_SECRET: test
+        run: |
+          # hardcode n because auto=2 in CI for some reason
+          just test-verification --migrations -n 4
```

Comment:
We probably don't need the `--migrations` flag here anymore.

Reply:
ah yes I missed that one - thanks!